test/src/Makefile

```diff
@@ -26,4 +26,4 @@ docker/test:
 .PHONY : clean
 ## Clean up files
 clean:
-	rm -rf $(TF_DATA_DIR) ../../examples/complete/*.tfstate*
+	rm -rf "$${TF_DATA_DIR}" ../../examples/complete/*.tfstate*
```

Comment:
@aknysh this really makes me uncomfortable. If someone, somehow sets `TF_DATA_DIR="/"` then this is big trouble. Can you perform some kind of test to make sure `TF_DATA_DIR` points to something safe?

Reply:
I don't see a reason for it to ever be in `test/src/Makefile`. If I put it there somewhere, please point it out to me, as I expect that was a mistake.

We sort of need it for the `clean` target at the top level Makefile for Geodesic, but really we should be using `atmos terraform clean` to make sure we get `TF_DATA_DIR` set, but then that doesn't work outside Geodesic.... smh

Reply:
Confirmed, it is a mistake from converting from very old to current testing framework, propagated by me by mistake. See 
- https://github.com/cloudposse/terraform-aws-vpc-flow-logs-s3-bucket/pull/58
- https://github.com/cloudposse/terraform-aws-s3-bucket/pull/181